### DISCUSSION TITLE
applying wal2json to database

### DISCUSSION THREAD

**Discussion by rajsinga** [eulerto/wal2json#227] [Q&A] [answered]
Is there a known way or tool to apply wal2json file to a database?
If not, is there a way to convert wal2json output to SQL statements?

**Comment by eulerto** [comment, marked as answer]
wal2json transforms data from the WAL internal representation into JSON format. If you want an SQL format, you need a different output plugin such as [decoder_raw](https://github.com/michaelpq/pg_plugins/tree/main/decoder_raw).
